edxval/api.py

```diff
@@ -733,6 +733,30 @@ def get_videos_for_course(course_id, sort_field=None, sort_dir=SortDirection.asc
     )
 
 
+@cachetools.func.ttl_cache(maxsize=None, ttl=settings.TTL_CACHE_TIMEOUT)
+def get_transcript_languages(course_id, provider_type):
+    """
```

Comment:
This method returns a language even if only one of the available course videos has a transcript in that language. Is this expected behavior?

Reply:
yup, it's expected. Output will be union of all transcript_languages with provider_type=`edx_ai_translations`